tests/backup_helper.go

```diff
@@ -9945,12 +9972,13 @@ func ValidatePartialRestore(ctx context1.Context, restoreName string, orgID stri
 		}
 
 		// VALIDATE APPLICATIONS
-		log.InfoD("Validate applications in restored namespace [%s] due to restore [%s]", expectedRestoredAppContextNamespace, restoreName)
-		errorChan := make(chan error, errorChannelSize)
-		ValidateContext(expectedRestoredAppContext, &errorChan)
-		for err := range errorChan {
-			errors = append(errors, err)
-		}
+		log.InfoD("Cannot validate applications in restored namespace [%s] due to restore [%s] "+
+			"because it is being restored from a partial backup and the apps may not be up and running", expectedRestoredAppContextNamespace, restoreName)
+		//errorChan := make(chan error, errorChannelSize)
+		//ValidateContext(expectedRestoredAppContext, &errorChan)
```

Comment:
Here , We can make use of appCtx.SkipPodValidation for namespace having failed volumes instead of skipping ValidateContext for entire namespaces.